code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -114,9 +102,8 @@
 
 	data["id_tag"] = -1 //must be defined in order to reuse code between portable and vent scrubbers
 	data["filter_types"] = list()
-	for(var/path in GLOB.meta_gas_info)
-		var/list/gas = GLOB.meta_gas_info[path]
-		data["filter_types"] += list(list("gas_id" = gas[META_GAS_ID], "gas_name" = gas[META_GAS_NAME], "enabled" = (path in scrubbing)))
+	for(var/path in GLOB.meta_gas_ids)
+		data["filter_types"] += list(list("gas_id" = path, "gas_name" = GLOB.meta_gas_names[path], "enabled" = (path in scrubbing)))
 
```

Comment:
Nother case of "has `path` changed"

Reply:
`path` in this case means the typepath of the gas datum, also referred to as the id in a lot of places.